src/toolkit/gis.cc

```diff
@@ -0,0 +1,158 @@
+#include "gis.h"
+
+using namespace std;
+
+namespace cyclus {
+namespace toolkit {
+
+GIS::GIS() : latitude_(0), longitude_(0) {}
+
+GIS::GIS(float decimal_lat, float decimal_lon) {
+  latitude_ = SetPrecision(decimal_lat * 3600, 1);
```

Comment:
3600 Should probably be a compile-time constant so it doesn't look like a magic number everywhere.